src/System.Security.Cryptography.Primitives/tests/CryptographicException.cs

```diff
@@ -17,7 +17,9 @@ public static void Ctor()
             Assert.NotNull(new CryptographicException().Message);
             Assert.Equal(message, new CryptographicException(message).Message);
             Assert.Equal(message + " 12345", new CryptographicException(message + " {0}", "12345").Message);
+#if !netfx
             Assert.Equal(5, new CryptographicException(5).HResult);
```

Comment:
Yeah, I'd just really like to be able to have everything expressed without \#if or other platform detection logic.